cmd/argocd/commands/app.go

```diff
@@ -705,7 +705,7 @@ func NewApplicationDiffCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 						errors.CheckError(err)
 						key = kube.GetResourceKey(target)
 					}
-					if key.Kind == "Secret" {
+					if key.Kind == kube.SecretKind && key.Group == "" {
```

Comment:
this is not a test, this is a code change

Reply:
Yes, this is minor improvement for previos PR